workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Table/Table.tsx

```diff
@@ -3,104 +3,136 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import ClayTable from '@clayui/table';
+import {
+	Body,
+	Cell,
+	Head,
+	Row as ClayRow,
+	Table as ClayTable,
+} from '@clayui/core';
 import {ClayTooltipProvider} from '@clayui/tooltip';
 import classNames from 'classnames';
 
+import './index.css';
 import TableColumn from '../../interfaces/tableColumn';
 
-import './index.css';
+type TableLayout = 'auto';
+
+interface BasicRow {
+	[key: string]: string | number | boolean | string[] | undefined;
+}
 
 interface TableProps<T> {
 	className?: string;
 	columns: TableColumn<T>[];
-	customClickOnRow?: (row: T) => void;
+	customClickOnRow?: (item: T) => void;
 	rows: T[];
+	tableLayout?: TableLayout;
 }
 
-const Table = <T extends unknown>({
-	className,
+interface RowProps<T> {
+	columns: TableColumn<T>[];
+	customClickOnRow?: (item: T) => void;
+	row: T;
+}
+
+type ChildrenRender<T> = ((item: T) => React.ReactElement) & string;
+
+const Row = <T extends BasicRow>({
 	columns,
 	customClickOnRow,
-	rows,
-}: TableProps<T>) => (
-	<ClayTooltipProvider>
-		<ClayTable
-			borderless
-			className={className}
-			noWrap
-			responsive
-			tableVerticalAlignment="middle"
+	row,
+}: RowProps<T>) => {
+	const id = Math.random().toString(16).slice(2);
+
+	return (
+		<ClayRow
+			className="border-0 font-weight-normal"
+			items={columns}
+			onClick={() => {
+				customClickOnRow && customClickOnRow(row);
```

Comment:
```suggestion
				if (customClickOnRow) {
                    customClickOnRow(row)
                }
```

This more readable to use if condition instead ternary